collect_app/src/main/java/org/odk/collect/android/utilities/TextUtils.java

```diff
@@ -86,14 +88,29 @@ private static String markdownToHtml(String text) {
         return text;
     }
 
-    public static CharSequence textToHtml(String text) {
+    public CharSequence textToHtml(String text) {
 
         if ( text == null ) {
             return null;
         }
 
-        return Html.fromHtml(markdownToHtml(text));
+        return htmlWrapper.toHtml(markdownToHtml(text));
+
+    }
+
+    public static class HtmlWrapper {
+        public Spanned toHtml(String text) {
+            return Html.fromHtml(text);
+        }
+    }
```

Comment:
This might be a good occasion to [continue the discussion](https://groups.google.com/forum/#!topic/opendatakit-developers/JNwzuuIlhrg) on our testing frameworks. This is the kind of thing I would use Robolectric for, but I'm not sure how the group feels about it. I'll post back there about it, as the answer will impact how Lindsay modifies the PR.